proposals/0251-font-styles.md

```diff
@@ -7,92 +7,73 @@
 
 ## Introduction
 
-This feature would let apps bold, underline or italicize text within Alert, Show or Perform Interaction RPCS.
+This feature would let apps style text within `Alert`, `Show`, `ScrollableMessage` or `PerformInteraction` RPC with things like bold, italics or underlining.
 
 ## Motivation
 
 As an app partner, I want to be able to customize text on main layouts and popups so that I can have a visually pleasing and easy to use UI.
 
 ## Proposed solution
 
-
-When an app wants a more visually appealing UI experience,  it can add a font style customization onto each line of text if desired.  If no font style is provided, then it is assumed that the font is normal.  
+When an app wants a more visually appealing UI experience,  it can add a HTML tags for font style customization onto text if desired.
```

Comment:
```suggestion
When an app wants a more visually appealing UI experience, it can add HTML tags for font style customization onto text if desired.
```